File: app/src/main/java/com/gwabs/ptds/activities/activities/MainActivity.java
package com.gwabs.ptds.activities.activities;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import androidx.appcompat.app.AppCompatActivity;

import com.gwabs.ptds.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    private ActivityMainBinding mainBinding;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        mainBinding = ActivityMainBinding.inflate(getLayoutInflater());
        View view = mainBinding.getRoot();
        setContentView(view);




        mainBinding.btnGetStarted.setOnClickListener(view1 -> {
            Toast.makeText(this, "you clicked me", Toast.LENGTH_SHORT).show();

            Intent intent = new Intent(this,Dashboard.class);
            startActivity(intent);
        });

    }
}

File: app/src/main/java/com/gwabs/ptds/activities/activities/MainActivity.kt
package com.gwabs.ptds.activities.activities

import android.content.Intent
import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.gwabs.ptds.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private var mainBinding: ActivityMainBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        mainBinding = ActivityMainBinding.inflate(layoutInflater)
        val view: View = mainBinding!!.root
        setContentView(view)
        mainBinding!!.btnGetStarted.setOnClickListener { view1: View? ->
            //Toast.makeText(this, "you clicked me", Toast.LENGTH_SHORT).show();
            val intent = Intent(this, Dashboard::class.java)
            startActivity(intent)
        }
    }
}